pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1613,9 +1613,9 @@ cdef class BusinessHour(BusinessMixin):
         Normalize start/end dates to midnight before generating date range.
     weekmask : str, Default 'Mon Tue Wed Thu Fri'
         Weekmask of valid business days, passed to ``numpy.busdaycalendar``.
-    start : str, default "09:00"
+    start : str, time, or list of str/time, default "09:00"
```

Comment:
Could you add to the `Examples` section showing this?

Reply:
@mroeschke Done